pkgs/development/tools/kubectx/default.nix

```
@@ -11,7 +11,11 @@ buildGoModule rec {
     sha256 = "sha256-WY0zFt76mvdzk/s2Rzqys8n+DVw6qg7V6Y8JncOUVCM=";
   };
 
-  vendorSha256 = "sha256-4sQaqC0BOsDfWH3cHy2EMQNMq6qiAcbV+RwxCdcSxsg=";
+  patches = [
+    ./bump-golang-x-sys.patch
```

Please submit this upstream.
```suggestion
    ./bump-golang-x-sys.patch
```

---

I did that but it won't apply because the go.mod changed otherwise since then in master https://github.com/ahmetb/kubectx/pull/360

---

Maybe just switch to using unstable/master?

---

I don't know, I leave that to the maintainer